### ISSUE TITLE
Change active tab underline color

### ISSUE THREAD

**Issue by adnxy** [react-native-community/react-native-tab-view#578]
### Current behaviour

When I switch between tabs, my underline color changes but it disappears when I'm done with pressing tab.

### Expected behaviour

I would like to have green underline color of active tab, that will stay there until next tab is clicked

### Code sample

  renderHeader = (props) => {
    return (<TabBar
      {...props}
      tabStyle={styles.tabBarStyle}
      labelStyle={styles.tabBarHeader}
      indicatorStyle={styles.tabBarIndicator}
      style={styles.tabBar}
      data={this.props}
      animationEnabled={false}
      pressColor={'green'}
    />);
  }

  renderScene = SceneMap({
    first: FirstRoute,
    second: SecondRoute,
    third: ThirdRoute,
  });

### Your Environment

| react-native          |
| react-native-tab-view |
| ios                  |
| npm      |

**Comment by jeprojects** [comment]
Is there an issue with styling the tab bar?

```
    <TabBar
      {...props}
      pressColor="rgba(179,229,255, 0.5)"
      scrollEnabled={true}
      style={styles.tabbar}
      indicatorStyle={styles.indicator}
      tabStyle={styles.tab}
      useNativeDriver
    />
```

In < 1.0.0 version this works for styling but now (1.0.0+) it gives the default blue background, white text etc.

**Comment by jungleyu** [comment]
It seems that the api changed from renderHeader to renderTabBar

**Comment by arirah** [comment]
Same here for me . Did you solve it ?

**Comment by satya164** [comment]
Hey, I just released a new alpha `2.0.0-alpha.0` of the library. It's rewritten using [react-native-gesture-handler](https://github.com/kmagiera/react-native-gesture-handler) and [react-native-reanimated](https://github.com/kmagiera/react-native-reanimated) addresses a many platform specific bugs and performance problems. The documentation is updated as well.

Please try the new version and see if it addresses your issue. If not, please open a new issue following the issue template.
